tests/Feature/Api/Resources/GetLanguageLinesByGroupTest.php

```diff
@@ -2,27 +2,22 @@
 
 namespace Tests\Feature\Api\Resources;
 
+use App\Enums\Language;
+use App\Enums\LanguageLineGroup;
 use App\Models\LanguageLine;
+use Illuminate\Support\Facades\App;
 use Tests\TestCase;
 
 /**
  * @internal
  *
  * @group Api\Resources
  *
- * @coversNothing
+ * @covers \App\Http\Controllers\Api\ResourcesController::getLanguageLinesByGroup
+ * @covers \App\Http\Resources\Models\LanguageLineResource::toArray
  */
 class GetLanguageLinesByGroupTest extends TestCase
 {
-    use DataProvider;
-
-    /**
-     * The method for the route endpoint.
-     *
-     * @var string
-     */
-    private const Method = 'GET';
```

Comment:
I don't see the need of a `::Method` const when the Resource itself states the it is a GET endpoint.
I would still use the `::Method` when the SUT doesn't make it clear.